masterlist.yaml

```
@@ -28237,9 +28237,9 @@ plugins:
         util: 'TES5Edit v4.0.1'
   - name: 'Karthwasten.esp'
     clean:
-      # version: 1.0.3
-      - crc: 0xC453CCC0
-        util: 'TES5Edit v4.0.1'
+      # version: 1.1.0
+      - crc: 0x035D37B0
+        util: 'TES5Edit v4.0.3'
```

I'll take that comment out, as it will be in the commit message anyway.
```suggestion
      - crc: 0x035D37B0
        util: 'TES5Edit v4.0.3'
```